scipy/optimize/_lsq/trf_linear.py

```diff
@@ -140,8 +140,8 @@ def select_step(x, A_h, g_h, c_h, p, p_h, d, lb, ub, theta):
         return ag
 
 
-def trf_linear(A, b, x_lsq, lb, ub, tol, lsq_solver, lsmr_tol, max_iter,
-               verbose):
+def trf_linear(A, b, x_lsq, lb, ub, tol, lsq_solver, lsmr_tol,
+               max_iter, verbose, lsmr_max_iter):
```

Comment:
As Tyler said.
```suggestion
              verbose, *, max_iter, lsmr_max_iter):
```

Reply:
Sorry, I missed this one; just fixed it. `lsmr_max_iter` is the only new argument I added, so that's the only one I made a keyword argument.